### PR TITLE
refactor: Move Location header logic to make_item_response

### PR DESCRIPTION
**Pull request by taion**

### CODE REVIEW

File: flask_resty/view.py [hunk 1]
```diff
@@ -183,11 +192,7 @@ def make_created_response(self, item):
         :return: The HTTP response
         :rtype: :py:class:`flask.Response`
         """
-        response = self.make_item_response(item, 201)
-        location = self.get_location(item)
-        if location is not None:
-            response.headers["Location"] = location
-        return response
+        return self.make_item_response(item, 201)
```
Comment (itajaja) [inline flask_resty/view.py:195]: you want to amend the doc of this method

Comment (taion) [reply flask_resty/view.py:195]: https://github.com/4Catalyzer/flask-resty/pull/328/files#diff-094ac497a5d94f5561a8a35fa2496623R101 does it, no?

Comment (taion) [reply flask_resty/view.py:195]: @itajaja does that answer the question, or am i missing something here?

Comment (itajaja) [reply flask_resty/view.py:195]: yeah of course 🤦‍♂ . all good

**Pull request merged**